docs/platform/understanding-airbyte/airbyte-protocol-docker.md

```diff
@@ -51,3 +51,25 @@ The `write` command will consume `AirbyteMessage`s from STDIN.
 - Connectors receive arguments on the command line via JSON files. `e.g. --catalog catalog.json`
 - They read `AirbyteMessage`s from STDIN. The destination `write` action is the only command that consumes `AirbyteMessage`s.
 - They emit `AirbyteMessage`s on STDOUT.
+
+## Additional Docker Image Requirements
+
+### Environment variable: `AIRBYTE_ENTRYPOINT` 
+
+The Docker image should contain an environment variable called `AIRBYTE_ENTRYPOINT`. This must be the same as the `ENTRYPOINT` of the image.
+
+## User: `Airbyte`
+
+The Docker image should run under a user named `airbyte`.
+
+## Specified `/airbyte` directory
+
+The Docker image must have a directory called `/airbyte`, which the user `airbyte` owns and can write to.
+
+This is the directory to which temporary files will be mounted, including the `config.json` and `catalog.json` files.
```

Comment:
I don't know if these are true.

Some connectors also have a `/airbyte/integrations` directory and I don't know if that is required or arbitrary.

Reply:
For platform, CHECK, SPEC, DISCOVER read and write in `/config`, while SYNC sources read / write from `/source` and destinations from `/dest`. I don't know what they do in connectors CI these days.